submissions/rubyAdeniregun/DOM/product.js

```diff
@@ -24,5 +24,79 @@ let submitForm1 = document.getElementById('verifyForm'),
         footeremailvalidation(e);
         return true;  
     }
-    
-    submitForm1.addEventListener("submit", verifyForm);
\ No newline at end of file
+
+submitForm1.addEventListener("submit", verifyForm);
+
+
+//EXAMPLE
+
+let item = document.getElementById("quantityBox")
+    button = document.getElementById("buyNow");
+    shippingCost = document.getElementById("shippingCost");
+    cost = document.getElementById("totalCost");
+    price = document.getElementById("price");
+
+//initial value
+let priceValue = price.innerHTML,
+    costValue = cost.innerHTML;
+//RegEx
+const Regex = /^[1-9]/;
+function calculateQuantity() {
+    if (!item.value.match(Regex)) {
+        button.disable = true
+        button.style = "background-color: grey"
+        item.nextElementSibling.innerHTML = "⚠️Please enter a valid quantity";
+    } else {
+        button.disabled = false;
+        item.nextElementSibling.innerHTML = "";
+    }
+}
+function calculatePrice() {
+    if (item.value >= 1) {
+        let perPrice = (2500 * item.value)
+        price.innerHTML = perPrice
+    } else if (item.value === "") {
+        price.innerHTML = priceValue
+        cost.innerHTML = costValue
+    }
+    else {
+        return false
+    }
+}
+function calculateShippingPrice() {
+    let perPrice = (2500 * item.value)
```

Comment:
There is no need for this `let perPrice = (2500 * item.value)` here. This calculation is already done inside the `calculatePrice()` function.
Let the `calculateShippingPrice()` take `perPrice` as a parameter ie. `calculateShippingPrice(perPrice)`. Then call the `calculateShippingPrice(perPrice)` inside the `calculatePrice()` function after the `let perPrice = (2500 * item.value)` calculation.